tests/lava/proc/conv_in_time/test_process.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2021-22 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import os
+import numpy as np
+from lava.proc.conv_in_time.process import ConvInTime
+from lava.proc import io
+
+from lava.magma.core.run_conditions import RunSteps
+from lava.magma.core.run_configs import Loihi1SimCfg
+
+if utils.TORCH_IS_AVAILABLE:
```

Comment:
This seems to error out
```bash
======================================================================
ERROR: tests.lava.proc.conv_in_time.test_process (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.lava.proc.conv_in_time.test_process
Traceback (most recent call last):
  File "/Users/runner/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/Users/runner/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/Users/runner/work/lava/lava/tests/lava/proc/conv_in_time/test_process.py", line 14, in <module>
    if utils.TORCH_IS_AVAILABLE:
NameError: name 'utils' is not defined


----------------------------------------------------------------------
```

You can either call `from lava.proc.conv import utils` or just use the saved ground truth and not use torch for verification at all.

Reply:
I added `from lava.proc.conv import utils`. Thanks for pointing out that.